tox.ini

```diff
@@ -34,8 +34,7 @@ norecursedirs = .* docs requirements
 
 [testenv]
 deps =
-    sphinx18: Sphinx==1.1.3
-    django19: Sphinx>=1.4,<1.5
+    Sphinx50: Sphinx>=5.0.0,<6.0.0
```

Comment:
The starting `s` of `Sphinx50` would be in `lowercase` just like here
https://github.com/openedx/edx-sphinx-theme/pull/199/files#diff-b803fcb7f17ed9235f1e5cb1fcd2f5d3b2838429d4368ae4c57ce4436577f03fR18